tests/handlers/test_koji.py

```diff
@@ -0,0 +1,75 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals
+
+import json
+from os import path
+from datetime import datetime
+
+from estuary.models.koji import KojiBuild, KojiTag
+from estuary.models.distgit import DistGitCommit
+import pytz
+import mock
+
+from tests import message_dir
+from estuary_updater.handlers.koji import KojiHandler
+from estuary_updater import config
+
+
+@mock.patch('koji.ClientSession')
+def test_build_complete(mock_koji_cs, mock_getBuild_two):
+    """Test the Koji handler when it recieves a new build complete message."""
+    mock_koji_session = mock.Mock()
+    mock_getBuild_two['state'] = 1
```

Comment:
This should actually be in the building state in Neo4j and then your handler marks it as complete.
You can do that with `koji.BUILD_STATES['BUILDING']`

Reply:
I made a mistake. I was thinking this was a build in Neo4j and not getBuild mock output.